### ISSUE TITLE
Q&A: propose questions/topics for the panel-discussion

### ISSUE THREAD

**Issue by DanielePucci** [loc2/ws_locomotion-manipulation#2]
Do not hesitate to comment this issue for adding questions you would like to be discussed during the `Panel discussion` at 15.15

**Comment by DanielePucci** [comment]
Possible discussion points.

- [ ] To which extent control and estimation techniques can be transferred from Aerial Manipulation to Humanoid Robotics?
- [ ] What are the hardware bottlenecks to extend the locomotion capabilities of humanoid robots to the flight case?

**Comment by pwensing** [comment]
- [ ] Is pairing with learning enough to address the existing barrier for online MPC in aerial or terrestrial locomotion? Is the barrier even there?

- [ ] What are the prospects for transporting the more formal treatment of control we observed in aerial manipulation talks to loco-manipulation? 

- [ ] To what degree might whole-body control strategies benefit aerial manipulation? What modifications might be necessary to address underactuations present?

- [ ] We saw that mixed-integer convex optimization could be used for provably optimal trajectory optimization for low-dimensional models. What are the practical limits of this approach scaling more broadly?
